quarkus-workshop-super-heroes/super-heroes/rest-fights/src/main/java/io/quarkus/workshop/superheroes/fight/FightResource.java

```diff
@@ -81,7 +80,20 @@ public Fight fight(@Valid Fighters fighters, UriInfo uriInfo) {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     @Path("/hello")
+    @Tag(name = "hello")
```

Comment:
I can't find anything in the adocs about this. Is it needed, or can we go simple?